pkg/controller/cronmanager.go

```diff
@@ -206,14 +207,7 @@ func (cm *CronManager) gcLoop() {
 
 // GC will collect all jobs which ref is not exists and recycle.
 func (cm *CronManager) GC() {
```

Comment:
GC func will be calling multiple times  when multiple cronhp CR be delete  at the same time. 
There will be many goruntine  running GC function at same time. 
Maybe GC function only need to be running only once in this case.